dali/pipeline/operators/reader/loader/sequence_loader.h

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2018, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_PIPELINE_OPERATORS_READER_LOADER_SEQUENCE_LOADER_H_
+#define DALI_PIPELINE_OPERATORS_READER_LOADER_SEQUENCE_LOADER_H_
+
+#include <numeric>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include "dali/common.h"
+#include "dali/pipeline/operators/reader/loader/loader.h"
+
+namespace dali {
+
+struct TensorSequence {
+  std::vector<Tensor<CPUBackend>> tensors;
+};
```

Comment:
There's this thing that we have: `Tensor`, `TensorList` and now `TensorSequence`. While I get that `Tensor` and `TensorList` could be separate data types (however, I don't feel like it), I think that `TensorSequence` and `TensorList` are basically the same. Is there any difference between these two data types? (Ofc, I'm asking about the concept point of view - as for implementation it's clear that TS is just a `vector` and TL is more sophisticated)
If conceptually it's the same, it should be put down to one data type...

Reply:
This is just the simple wrapper for some set of `Tensor`s that are loaded by the SequenceLoader`. Similar to `ImageLabelWrapper` for `FileReader`, that stores `Tensor` for image data and `int` for appropriate label. Maybe the naming of those should follow some other convention.

Currently I want to have Sequence represented as highest dimension of regular `Tensor`, but in this case, before "frames" are decoded, they may have different sizes, so we cannot do that.